pkg/sessiontxn/txn_context_test.go

```diff
@@ -505,12 +505,12 @@ func TestTxnContextForPrepareExecute(t *testing.T) {
 	doWithCheckPath(t, se, normalPathRecords, func() {
 		tk.MustExec("prepare s from 'select * from t1 where id=1'")
 	})
-	doWithCheckPath(t, se, normalPathRecords, func() {
+	doWithCheckPath(t, se, []string{"assertTxnManagerInCompile", "assertTxnManagerInShortPointGetPlan"}, func() {
 		tk.MustQuery("execute s").Check(testkit.Rows("1 10"))
 	})
 
 	// Test ExecutePreparedStmt
-	doWithCheckPath(t, se, normalPathRecords, func() {
+	doWithCheckPath(t, se, []string{"assertTxnManagerInCompile", "assertTxnManagerInShortPointGetPlan"}, func() {
```

Comment:
Expected, these 2 cases should be able to reuse PointGet executors

Reply:
It's their first time execution, they are not reusing executors.
Previously, it won't go into `(a *ExecStmt) PointGet` for the first execution, now it will. Is that expected?